data_handling/NN_control_node.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun 28 09:04:17 2017
+
+@author: Zorn, Ronecker
+"""
+
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 30 17:18:42 2017
+
+@author: ben
+"""
+
+import numpy as np
+import rospy
+import os
+import sys
+import h5py
+import datetime
+import cv2
+import message_filters
+
+from torcs_msgs.msg import TORCSSensors
+from sensor_msgs.msg import LaserScan
+from sensor_msgs.msg import Image
+from std_srvs.srv import Trigger
+from torcs_msgs.msg import TORCSCtrl
+from cv_bridge import CvBridge, CvBridgeError
+from std_msgs.msg import String
+import keras
+from keras.models import load_model
+import tensorflow as tf
+
+
+
+
+
+class Controler:
+    def __init__(self, image_topic='torcs_ros/pov_image',control_topic='/torcs_ros/ctrl_cmd', sensors_topic='/torcs_ros/sensors_state'):
+        self.model_path = '/home/max/Schreibtisch/weights_epoch_99.h5' #load neural net at path

```

Comment:
@B-Zorn please make the model path a parameter instead of hard coding it here! Will you upload an example of one trained model to the LRZ shared drive or add it to this PR (depending on size of the file)?

Reply:
I'll upload a trained model to the LRZ drive

Reply:
Awesome, thanks for your effort :smiley: